2021-01-16 16:13:46,253 INFO    MainThread:2201308 [wandb_setup.py:_flush():70] setting env: {}
2021-01-16 16:13:46,253 INFO    MainThread:2201308 [wandb_setup.py:_flush():70] multiprocessing start_methods=fork,spawn,forkserver
2021-01-16 16:13:46,253 INFO    MainThread:2201308 [wandb_setup.py:_flush():70] setting login settings: {}
2021-01-16 16:13:46,253 INFO    MainThread:2201308 [wandb_init.py:_log_setup():318] Logging user logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_161346-q4dqtm8c/logs/debug.log
2021-01-16 16:13:46,254 INFO    MainThread:2201308 [wandb_init.py:_log_setup():319] Logging internal logs to /home/whiteashes/Project/FedML/fedml_experiments/standalone/fedavg/wandb/run-20210116_161346-q4dqtm8c/logs/debug-internal.log
2021-01-16 16:13:47,909 INFO    MainThread:2201308 [wandb_run.py:_console_start():1308] atexit reg
2021-01-16 16:13:47,910 INFO    MainThread:2201308 [wandb_run.py:_redirect():1175] redirect: SettingsConsole.REDIRECT
2021-01-16 16:13:47,910 INFO    MainThread:2201308 [wandb_run.py:_redirect():1178] Redirecting console.
2021-01-16 16:13:47,910 INFO    MainThread:2201308 [redirect.py:install():205] install start
2021-01-16 16:13:47,910 INFO    MainThread:2201308 [redirect.py:install():220] install stop
2021-01-16 16:13:47,910 INFO    MainThread:2201308 [redirect.py:install():205] install start
2021-01-16 16:13:47,910 INFO    MainThread:2201308 [redirect.py:install():220] install stop
2021-01-16 16:13:47,910 INFO    MainThread:2201308 [wandb_run.py:_redirect():1222] Redirects installed.
2021-01-16 16:14:03,325 INFO    MainThread:2201308 [wandb_run.py:_atexit_cleanup():1278] got exitcode: 1
2021-01-16 16:14:03,326 INFO    MainThread:2201308 [wandb_run.py:_restore():1250] restore
2021-01-16 16:14:03,326 INFO    MainThread:2201308 [redirect.py:uninstall():224] uninstall start
2021-01-16 16:14:03,326 INFO    MainThread:2201308 [redirect.py:_stop():279] _stop: stdout
2021-01-16 16:14:03,326 INFO    MainThread:2201308 [redirect.py:_stop():285] _stop closed: stdout
2021-01-16 16:14:03,326 INFO    stdout    :2201308 [redirect.py:_pipe_relay():121] relay done saw last write: stdout
2021-01-16 16:14:03,326 INFO    stdout    :2201308 [redirect.py:_pipe_relay():137] relay done done: stdout
2021-01-16 16:14:03,326 INFO    MainThread:2201308 [redirect.py:_stop():291] _stop joined: stdout
2021-01-16 16:14:03,326 INFO    MainThread:2201308 [redirect.py:_stop():293] _stop rd closed: stdout
2021-01-16 16:14:03,326 INFO    MainThread:2201308 [redirect.py:uninstall():228] uninstall done
2021-01-16 16:14:03,327 INFO    MainThread:2201308 [redirect.py:uninstall():224] uninstall start
2021-01-16 16:14:03,327 INFO    MainThread:2201308 [redirect.py:_stop():279] _stop: stderr
2021-01-16 16:14:03,327 INFO    MainThread:2201308 [redirect.py:_stop():285] _stop closed: stderr
2021-01-16 16:14:03,327 INFO    stderr    :2201308 [redirect.py:_pipe_relay():121] relay done saw last write: stderr
2021-01-16 16:14:03,327 INFO    stderr    :2201308 [redirect.py:_pipe_relay():137] relay done done: stderr
2021-01-16 16:14:03,327 INFO    MainThread:2201308 [redirect.py:_stop():291] _stop joined: stderr
2021-01-16 16:14:03,327 INFO    MainThread:2201308 [redirect.py:_stop():293] _stop rd closed: stderr
2021-01-16 16:14:03,327 INFO    MainThread:2201308 [redirect.py:uninstall():228] uninstall done
2021-01-16 16:14:03,773 INFO    MainThread:2201308 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "d470b2030529451d8647605a55774d8c"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1157
      total_bytes: 1157
    }
  }
}

2021-01-16 16:14:05,776 INFO    MainThread:2201308 [wandb_run.py:_wait_for_finish():1395] got exit ret: uuid: "e25e8448395447879affbd95841adc27"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 5398
      total_bytes: 5398
    }
  }
}

2021-01-16 16:14:06,825 INFO    MainThread:2201308 [wandb_run.py:_show_files():1614] logging synced files
2021-01-16 16:14:06,826 INFO    MainThread:2201308 [internal.py:handle_exit():139] Internal process exited
